lib/helpers.py

```diff
@@ -10,6 +10,40 @@
 
 cache_folder = "cache-v{0}".format(__version__)
 
+# columns to load from CSV
+bid_columns = ['ts', 'user_id', 'ab_test_group', 'campaign_id','cost_eur','event_type']
+attribution_columns = ['ts', 'user_id', 'partner_event', 'revenue_eur']  
+
+
+def load_marks_and_spend_data(customer, audiences, dates):
```

Comment:
Please, for the future changes, split refactorings (like this one) into separate commits from actual changes.